akka-cluster-sharding/src/test/scala/akka/cluster/sharding/LeastShardAllocationStrategySpec.scala

```diff
@@ -18,7 +18,7 @@ class LeastShardAllocationStrategySpec extends AkkaSpec {
   "LeastShardAllocationStrategy" must {
     "allocate to region with least number of shards" in {
       val allocations = Map(regionA -> Vector("shard1"), regionB -> Vector("shard2"), regionC -> Vector.empty)
-      allocationStrategy.allocateShard(regionA, "shard3", allocations) should ===(regionC)
+      allocationStrategy.allocateShard(regionA, "shard3", allocations).value.get.get should ===(regionC)
```

Comment:
yes, and `LeastShardAllocationStrategy` is supposed to be completed immediately. I can change to Await, though.